Star Practices/src/textSearch/InvertedIndex.java

```diff
@@ -15,6 +33,9 @@ public class InvertedIndex {
     public InvertedIndex(List<Pair<String, Integer>> tokens, int fileCount) {
```

Comment:
The constructor does too many jobs, and violates the Single Responsibility principle. Try to use method extraction.